week-1/exercises/random.js

```diff
@@ -2,8 +2,30 @@ const minimum = 1;
 const maximum = 100;
 
 const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
+// console.log(num);
```

Comment:
Generally, committing code in a comment isn't a good practice. (You'll often see developers commenting out old code, when instead we can use tools like Git to find old code if it's needed. Good practice is to completely remove old code.)

Reply:
I don't understand Jay, how can I use Git to do the task?